libs/common/backend/models/README.md

```diff
@@ -0,0 +1,7 @@
+# common-backend-models
+
+This library was generated with [Nx](https://nx.dev).
+
+## Running unit tests
+
+Run `nx test common-backend-models` to execute the unit tests via [Jest](https://jestjs.io).
```

Comment:
Some docs would be good, I guess not all project members have used DDD yet. Therefore a description might be helpful for their understanding.

Reply:
Did so :)

Reply:
Perfect